x/programs/rust/wasmlanche-sdk/src/context.rs

```diff
@@ -0,0 +1,129 @@
+use crate::{
+    state::{Error, Key, State},
+    types::Address,
+    Gas, Id, Program,
+};
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::{cell::RefCell, collections::HashMap};
+
+/// Representation of the context that is passed to programs at runtime.
+#[cfg_attr(feature = "debug", derive(Debug))]
+pub struct Context<K = ()> {
+    program: Program,
```

Comment:
could change the `program` field to just be an address?

Reply:
Will do in a follow up